### DISCUSSION TITLE
The css variable should not take up the style attribute of the html

### DISCUSSION THREAD

**Discussion by Pszz** [WalletConnect/walletconnect-monorepo#3342] [Ideas]
<img width="1119" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/12093551/f851b982-2382-433f-83e1-79653adc556f">

I don't think these css variables should be inside the html style attribute.

First, it may override the style attribute of the project html itself.
Second, it's very unattractive and looks like dirty code.

Maybe it's better this way.

```css
:root {
  --w3m-color-fg-1: rgb(20,20,20);
}
```

or

```css
html {
  --w3m-color-fg-1: rgb(20,20,20); 
}
```

**Comment by boidushya** [comment]
Upcoming Web3Modal v3 takes care of this 🤝 
Keep an eye out 
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/26746725/9e044573-8da3-4112-9944-7246c17e385d)
